.gitignore

```diff
@@ -1,3 +1,14 @@
+*.iml
```

Comment:
the .gitignore of a project should only contains specific ignores of these project. I can see into this file ignores from a specific IDE that you are using (intellij). I think this ignores should be in your global .gitignore file, because is project-agnostic.